modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/migration/CaseInstanceMigrationManagerImpl.java

```diff
@@ -256,6 +256,10 @@ public void migrateHistoricCaseInstancesOfCaseDefinition(String caseDefinitionId
 
     protected void doMigrateCaseInstance(CaseInstanceEntity caseInstance, CaseDefinition caseDefinitionToMigrateTo, CaseInstanceMigrationDocument document, CommandContext commandContext) {
         LOGGER.debug("Start migration of case instance with Id:'{}' to case definition identified by {}", caseInstance.getId(), printCaseDefinitionIdentifierMessage(document));
+        if(document.getPreUpgradeExpression() != null && !document.getPreUpgradeExpression().isEmpty()){
```

Comment:
```suggestion
        if (document.getPreUpgradeExpression() != null && !document.getPreUpgradeExpression().isEmpty()) {
```
Typically the project uses spaces here